app/serializers/stripe-connect-account.js

```diff
@@ -4,5 +4,15 @@ export default ApplicationSerializer.extend({
   attrs: {
     legalEntitySsnLast4: { key: 'legal-entity-ssn-last-4' },
     legalEntitySsnLast4Provided: { key: 'legal-entity-ssn-last-4-provided' }
-  }
+  },
+
+  /**
+   * Overrides default serializeAttributes so it only serializes changed attributes,
+   * instead of the default behavior, which is all of them.
+   */
+  serializeAttribute(snapshot, json, key) {
+    if (snapshot.changedAttributes()[key] || snapshot.record.get('isNew')) {
+      this._super(...arguments);
+    }
+  },
```

Comment:
Should this be our default serialization behavior everywhere? I feel like it makes more sense than constantly resending values.

Of course, there may be `required` parameters regardless of whether they have changed, so perhaps this is untenable.